sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -22,7 +22,7 @@ public class TextAnalyticsClient
         internal readonly TextAnalyticsRestClient _serviceRestClient;
         internal readonly ClientDiagnostics _clientDiagnostics;
         private readonly TextAnalyticsClientOptions _options;
-        private readonly string DefaultCognitiveScope = "https://cognitiveservices.azure.com/.default";
+        private readonly string DefaultCognitiveScope = TextAnalyticsClientOptions.getDefaultCognitiveScope(Environment.GetEnvironmentVariable("TENANT_ID"));
```

Comment:
Look at the `TENANT_ID` was also my first choice, it looked good in the local test, but it could not get `EnvironmentVariable: TENANT_ID` in Pipeline. So i change the way and get `AZURE_AUTHORITY_HOST` for Form Recognizer. I will remove these changes in Text Analytics, because of `Healthcare` tests are failing in per clouds except Public.

Reply:
Hmm interesting. Getting `TENANT_ID` should always work as we always need it when running our test with AAD. See how our `TestEnvironment` does it [here](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs#L121).

As for Text Analytics, yeah, `Healthcare` only works on public. This might change in a couple of months though

Reply:
I am strange for it. Was the value of `TENANT_ID` constant in every clouds? Maybe I set it's value wrong.
`Public : 72f988bf-86f1-41af-91ab-2d7cd011db47`
`UsGov  : 63296244-ce2c-46d8-bc36-3e558792fbee`
`China  : 3d0a72e2-8b06-4528-98df-1391c6f12c11`